client/my-sites/upgrades/checkout-thank-you/footer/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Sounds good, let's just not let it sit idle since this makes the whole system more fragile :)